Automattic/newspack-plugin
# Fix/donations processing

### All Submissions:






### Changes proposed in this Pull Request:



Fixes a small issue with donations. Also, prevents bailing from donation handling if `Reader_Activation::register_reader` returns an error. 

### How to test the changes in this Pull Request:

1. On `master`, set up a new site or remove Donations products
2. Activate WooCommerce, but not the other WooCommerce plugins 
3. Configure site to use Stripe as Reader Revenue platform
4. Make a donation, observe a PHP warning logged
5. Switch to this branch, observe donation going through without issues

### Other information:
